mailpoet/assets/js/src/wizard/steps/pitch_mss_step/second_part.tsx

```diff
@@ -42,7 +42,7 @@ function MSSStepSecondPart({
               key={index}
               target="_blank"
               rel="noreferrer"
-              href="https://account.mailpoet.com/"
+              href="https://account.mailpoet.com/?ref=plugin-wizard&utm_source=plugin&utm_medium=onboarding&utm_campaign=purchase"
```

Comment:
I remember this discussion as well but like you I'm not able to find it. I have a vague memory that the conclusion to not use it because it is not currently working properly, but I could be wrong. Checking the service quickly, I was not able to find a method that I could use to generate the same URL that I'm generating here.

Reply:
I found it ([#](https://mailpoet.atlassian.net/browse/MAILPOET-4749?focusedCommentId=23366))! This time I searched using the name of the service. It seems we don't need to use it. So we are fine. Maybe we could get rid of it. I'll ask on slack.